packages/arb-token-bridge-ui/src/token-bridge-sdk/EthWithdrawalStarter.ts

```diff
@@ -59,12 +64,25 @@ export class EthWithdrawalStarter extends BridgeTransferStarter {
       throw 'Native currency withdrawals to a custom destination address are not supported yet.'
     }
 
+    await validateSignerChainId({
```

Comment:
Everywhere - We should also add a check for `addressIsSmartContract` along with this.

+ We can abstract `validateSignerChainId + addressIsSmartContract = validateTransfer (or something like that)` as 1 function which takes in the toAddress+signer as input and does all the checks within it.